lambdas/enums/metadata_field_names.py

```diff
@@ -0,0 +1,33 @@
+from enum import Enum
+
+
+class DynamoField(Enum):
```

Comment:
This Class is overlapping with the NHSDocumentReference class. Can you merge them into one?

Reply:
I'd argue that this class is a reference for querying Dynamo, wheras NHSDocumentreference is a data object used for uploading data. However, if you have an idea on how to combine these two cleanly, I'd be happy to have a chat about it

Reply:
Spoke to Noga and sorted 👍